README.md

```diff
@@ -0,0 +1,58 @@
+# TechJobs, MVC Edition
+
+For [this assignment](https://education.launchcode.org/csharp/assignments/tech-jobs-mvc.html) students are asked to once again refactor a TechJobs application, this time using an MVC 
+design pattern.
+
+## Assignment Requirements
+
+1. Review Starter Code
+2. Complete `ListController`
+3. Complete `SearchController`
+4. Display Search Results
+
+## Unit Tests
+
+The starter code contains 4 unit test classes.  As with previous graded assignments, each test class was created for a specific task in the instructions.
+
+`TestTaskTwo` verifies the work done to complete the List Views.  This includes displaying a list of jobs.
+
+`TestTaskThree` verifies work done to complete the `SearchController`.  This includes verifying how the `Results()` method was created and making sure that the method uses local variables. This test class focuses on checking your implementation.
+
+`TestTaskFour` verifies the behaviors required for `Results()` to pass accurate data to a view.
+ 
+### Appearance and Code Check:
+ 
+[Working Demo App](https://csharp-mvc.launchcodetechnicaltraining.org/)
```

Comment:
https://csharp-mvc.launchcodetechnicaltraining.org/. I'm getting an error when I click on Search or List, is that suppose to happen?

Reply:
Please add verbiage about why students may run into issues with the app.

Reply:
Done!